### PR TITLE
182 receive peers and block information from node

### PR DESCRIPTION
**Pull request by isSerge**
- remove connection to Public RPC node
- remove segments and blockchain size - show synced blocks instead
- mnemonic modal clean up

### CODE REVIEW

File: src/pages/Dashboard.vue [hunk 1]
```diff
@@ -76,42 +76,34 @@ export default defineComponent({
...
-          util.errorLogger("DASHBOARD | node name was empty when tried to start node")
-        }
-        util.infoLogger("DASHBOARD | starting farmer")
-        const farmerStarted = await this.$client.startFarming(config.plot.location, config.plot.sizeGB)
-        if (!farmerStarted) {
-          util.errorLogger("DASHBOARD | Farmer start error!")
-        }
-        util.infoLogger("DASHBOARD | starting block subscription")
-        await this.$client.startBlockSubscription()
+    if (this.$client.isFirstLoad() === false) {
```
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply src/pages/Dashboard.vue:79]: I know. Maybe i explained it bad. 
Let me rephrase:
I can't see `connectApi` in dashboard, but in order to use `CheckNodeAndNetwork`, i think a connection to the `api` must be established. 
Does this even work?

File: src/pages/PlottingProgress.vue [hunk 2]
```diff
@@ -229,23 +223,23 @@ export default defineComponent({
...
-      this.networkSegmentCount = networkSegmentCount
+
       this.plottingData.allocatedGB = config.plot.sizeGB
-      this.localSegmentCount = await this.$client.getLocalSegmentCount()
-      do {
-        await new Promise((resolve) => setTimeout(resolve, 2000))
-        this.localSegmentCount = await this.$client.getLocalSegmentCount()
-      } while (this.localSegmentCount < this.networkSegmentCount)
 
+      this.syncState = (await this.$client.getSyncState()).toJSON() as unknown as SyncState;
```
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply src/pages/PlottingProgress.vue:229]: If we are going to refactor it, i think its fine. 
As Nazar suggested, maybe we can have `TODOs` in such places to remind ourselves and the reviewer that it will be refactored. We can use that approach in the next PR's

File: src/pages/Dashboard.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (isSerge) [reply src/pages/Dashboard.vue:79]: please check 'waitNodeStartApiConnect' method, this is the one called from Dashboard and Plotting pages

File: src/pages/PlottingProgress.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (isSerge) [reply src/pages/PlottingProgress.vue:229]: I want to clarify how far we're ready to go with the refactoring of this version (before we switch to React) at the meeting tomorrow. Otherwise, I would have to add too many TODOs

**Review by ozgunozerk — APPROVED** [review]

**Review by nazar-pc — APPROVED** [review]
Looks fine to me
